app/src/main/java/com/framgia/moviedbtraining/MainActivity.kt

```diff
@@ -59,51 +62,60 @@ class MainActivity : AppCompatActivity() {
       fragmentTransaction.commitAllowingStateLoss()
     }
     mHandler!!.post(mPendingRunnable)
-    drawer_layout.closeDrawers()
+    mDrawerLayout.closeDrawers()
     invalidateOptionsMenu()
   }
 
   private fun getMoviesFragment(): Fragment {
     when (navItemIndex) {
-      TAG_MOVIE -> {
-        return MovieFragment()
+      TAG_NOW -> {
```

Comment:
all fragment is movieFragment why can not return movieFragment.newinstance(navItemIndex) ???